sections/06_discussion.md

```diff
@@ -6,22 +6,10 @@ can be further pruned.*
 
 ### Evaluation
 
-*What are the challenges in evaluating deep learning models that are specific to
-this domain?  This can include a discussion of ROC versus precision-recall
-curves for the imbalanced classes often encountered in biomedical datasets.
-It could also mention alternative metrics that are used in specific sub-areas
-such as enrichment factors in virtual screening.  A lack of true gold standard
-data for some problems complicates both training and evaluation.  Confidence-
```

Comment:
This part of the prompt came from @akundaje in #109:

> Yes confidence rated predictions are important and also in gold standard datasets providing "ambiguous labels" or confidence on labels is useful and a good practice. It helps with evaluation (you can weight the loss by the confidence in the label) as well.

I'd still like to include that point here.